libs/shared/running-tasks/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "version": "0.0.1",
+  "private": true,
+  "name": "@nx-console/shared-running-tasks",
+  "main": "./src/index.ts",
+  "dependencies": {
+    "tslib": "^2.3.0"
+  },
+  "nx": {
```

Comment:
nit but other libs have an nx name without the scope, so `shared-running-tasks` in this case